File: Patterns/printPattern7.cpp
#include <iostream>
using namespace std;
void pattern7(int row){
    int nst=1;
    int nsp=0;
    for(int r=1; r<=row; r++){
        for(int csp=1; csp<=nsp; csp++){
            cout<<"\t";
        }
        for(int cst=1; cst<=nst; cst++){
            cout<<"*\t";
        }
        
        nsp++;
        cout<<endl;
    }
}
int main(int argc, char **argv){
    int n;
    cin >> n;
    pattern7(n);
    //write your code here
    
}

File: Patterns/printPattern7.java
import java.util.*;

public class Main {
    public static void printPattern7(int row) {
        for(int i = 1; i <= row; i++) {
            for(int j = 1; j <= row; j++) {
                if(i == j) {
                    System.out.print("*\t");
                } else {
                    System.out.print("\t");
                }
            }
            System.out.println();
        }
    }

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int row = scn.nextInt();
        printPattern7(row);
        // write ur code here

    }
}